docs/patterns/how-to-maintain-key-vaults-IaC.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: pattern
+order: 4
+title: How to maintain secrets with key vaults IaC
+date: 2025-11-06 # this should be the date that the content was most recently amended or formally reviewed
+# use `tags: []` for no tags 
+# Note: tags must use sentence case capitalisation
+tags:
+  - Security
+  - Digital
+  - IaC
+related: # remove this section if you do not need related links on your page
+  sections:
+    - title: Related Standards
+      items:
+        - text: Secret Management
+          href: /standards/secret-management/ # Note: use an absolute link from the site home page
+        - text: IaC Bicep Configuration
+          href: /standards/standard-iac-bicep-configuration/ 
+---
+
+<!-- Pattern description -->
+
+<!-- 
+
+# Notes on line breaks
+
+Please see https://x-govuk.github.io/govuk-eleventy-plugin/markdown/#line-breaks for notes on usage of line breaks.
+
+# Notes on linking to headings within a page
+
+Heading tags are automatically assigned an id, converting spaces to `kebab-case` and applying URL encoding. If you want to link to a specific heading, you can obtain the URL encoded link by running the site locally, inspecting the appropriate <h3> element in the browser's developer tools and copying the value from the 'id' attribute.
+-->
+
```

Comment:
Probably worth starting out with a description for what the patterns is intended for